compiler/src/main/scala/org/scalajs/core/compiler/PrepJSInterop.scala

```diff
@@ -1332,7 +1332,7 @@ abstract class PrepJSInterop[G <: Global with Singleton](val global: G)
 
       val firstArgIsValid = annot.stringArg(0).isDefined
       if (!firstArgIsValid) {
-        reporter.error(annot.args.head.pos,
+        reporter.error(annot.args(0).pos,
```

Comment:
This commit reverted the wrong change. It should have reverted the change in `OptimizerCore.scala`, not in `PrepJSInterop.scala`.

Reply:
Oops. 
Fixed in https://github.com/scala-js/scala-js/pull/3653/commits/fa30b61aa1aa50090db5d9cc69319dd31e0237d8

Reply:
Oops. 
Fixed in https://github.com/scala-js/scala-js/pull/3653/commits/e108cb42e9e360b6e352d24d167f258cbe1c1e8e

Reply:
Oops. 
Fixed in https://github.com/scala-js/scala-js/pull/3653/commits/67dbe0df56927fa0251ea342165d9b82a1980056

Reply:
Oops.
Fixed in https://github.com/scala-js/scala-js/pull/3653/commits/e43e0fb30cba80a23e8a996f535fe9376a4a7b5c